Dockerfile

```
@@ -0,0 +1,36 @@
+FROM python:3.8-slim
+RUN apt update && apt install -y wget git libglib2.0-0 libsm6 libxext6 libxrender-dev
+RUN apt clean && rm -rf /var/lib/apt/lists/*
+# RUN useradd -ms /bin/bash stablediff
+# install conda
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && mkdir /root/.conda \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh \
+    && echo PATH="/root/miniconda3/bin":$PATH >> .bashrc 
+RUN chmod +x /root/miniconda3/bin/conda
+RUN ln -s /root/miniconda3/bin/conda /usr/local/bin/conda
+RUN conda update -y conda
+# i am using git clone instead during development of this dockerfile
+COPY . /app/ 
+RUN mkdir /app/outputs/
+RUN mkdir /app/weights/
+# RUN git clone https://github.com/CompVis/stable-diffusion.git /app/
+WORKDIR /app/
+RUN conda env create -f /app/environment.yaml -n ldm
```

Redundant absolute path after setting `WORKDIR` to `/app`
```suggestion
RUN conda env create -f environment.yaml -n ldm
```